extmod/uasyncio/stream.py

```diff
@@ -80,7 +80,8 @@ async def open_connection(host, port):
     import usocket as socket
 
     ai = socket.getaddrinfo(host, port)[0]  # TODO this is blocking!
-    s = socket.socket()
+    addr_family = ai[0]
+    s = socket.socket(addr_family)
```

Comment:
I think it makes sense to do it like `tools/upip.py` does it (and elsewhere in micropython-lib):
```python
ai = socket.getaddrinfo(host, port, 0, socket.SOCK_STREAM)
s = socket.socket(ai[0], ai[1], ai[2])
```

Reply:
Awesome. Applied changes.